README.md

```diff
@@ -15,14 +15,22 @@
 
 ### Compiling from Source
 
-If you prefer to compile from source or are contributing to the project, you can install `kuadrantctl` using `go install`. This method requires Golang 1.21 or newer.
+If you prefer to compile from source or are contributing to the project, you can install `kuadrantctl` using `go install` or `make install`. This method requires Golang 1.21 or newer.
 
 ```bash
 go install github.com/kuadrant/kuadrantctl@latest
 ```
 
 This command will compile `kuadrantctl` and install the binary executable in `$GOBIN` (defaulting to `$GOPATH/bin`).
 
+It is also possible to use the make target `install` to compile from source. From root of the repository, run 
+
+```bash
+make install
+```
+
+This will compile `kuadrantctl` and install it in the `bin` directory at root of directory. It will also ensure the correct version of the binary is displayed, rather than `v0.0.0` . It can be ran using `./bin/kuadrantctl` .  
```

Comment:
I would not mention the "v0.0.0" if it was not used in the end.

Reply:
> go build creates binary which prints empty version

Yes, you're right! I focused on creating the make target which has the parameters for displaying the version, which displays nothing if just `go build` was ran. A solution to this problem might be to have `v0.0.0` displayed when compiling this way.

> make install without git configured creates binary with version "dev -"

That's right - relying on an external dependency such as git can present issues if git is not installed on the system. An alternative might be when the make target is ran, it checks to see if git is installed, and if not, doesn't touch `version/version.go` at all, and just compiles.

> make install with git configured creates binary with correct version with commit hash

This is the expected outcome, which is what we want.

> I would somehow merge the 1. and 2., maybe setting a default value in version/version.go and not in Makefile?

Yes, I think you're right! I will go with having a default value in `version/version.go` . Thanks for suggesting this change!